client/my-sites/site-settings/form-writing.jsx

```diff
@@ -134,19 +135,21 @@ class SiteSettingsFormWriting extends Component {
 							isRequestingSettings={ isRequestingSettings }
 							fields={ fields }
 							jetpackVersionSupportsLazyImages={ jetpackVersionSupportsLazyImages }
+							siteAcceleratorSupported={ siteAcceleratorSupported }
```

Comment:
It seems like `siteAcceleratorSupported` isn't used in `<MediaSettings />` at all.